cron.php

```diff
@@ -43,9 +43,11 @@
 
 umask(0);
 
-$disabledFuncs = array_map('trim', explode(',', strtolower(ini_get('disable_functions'))));
-$isShellDisabled = is_array($disabledFuncs) ? in_array('shell_exec', $disabledFuncs) : true;
-$isShellDisabled = (stripos(PHP_OS, 'win') === false) ? $isShellDisabled : true;
+$disabledFuncs = array_map('trim', preg_split("/,|\s+/", strtolower(ini_get('disable_functions'))));
+$isShellDisabled = in_array('shell_exec', $disabledFuncs)
+    || !str_contains(strtolower(PHP_OS), 'win')
```

Comment:
`str_contains` is only available starting from PHP 8

Reply:
There is a polyfill for that, I made sure before proposing it.

Reply:
ahh my bad then

Reply:
I'm on PHP 7.4 and currently it is showing the following error: PHP Fatal error:  Uncaught Error: Call to undefined function str_contains() in cron.php:46

Reply:
@git-seb Do you have any custom modifications done on your `cron.php`? I just ran it now with PHP 7.4 and it's working fine...

Reply:
Also, can you check your `app/code/core/Mage/Core/functions.php` matches the content from the release you're using if you did the update manually.